src/core/types.ts

```diff
@@ -78,15 +79,18 @@ export interface PolygonContracts extends BothContracts {
 export interface EthContracts extends BothContracts {
   ethEarn: Vault;
   ethWethEarn: Vault;
-  weth: ethers.Contract;
   ssvEthUSDEarn: StrategyVault;
   withdrawalEscrow: WithdrawalEscrow;
   router: Router;
   degen: Vault;
   ethLeverage?: Vault;
 }
 
-export interface AlpineContracts extends PolygonContracts, EthContracts {}
+export interface BaseContracts extends BothContracts {
+  baseUsdEarn: Vault;
```

Comment:
Same as before. But, It is okay if we deploy the vault all across the envs like, testnet, mainnet & forked mainnet